pkg/utils/utils.go

```diff
@@ -56,6 +59,9 @@ var (
 	// ReqMountList contains the list of volumes which are required
 	// to be remounted. This list is secured by ReqMountListLock
 	ReqMountList map[string]apis.CStorVolumeAttachmentStatus
+
+	// should send google analytics or not
+	GoogleAnalyticsEnabled string
```

Comment:
n_p: we can avoid exporting it and write setter and getter methods/functions instead. 
Reason: exposing variables that should not be meant to be modified by somebody else should be ensured as far as possible

Reply:
Now not storing in a Global const , instead getting and verifying the env on every send call

Reply:
getter might have been better approach as the ENV isn't going to change in application life-cycle and it lacks the functionality to toggle it in run-time(without restarting it).